tests/integration_tests/data_steward/validation/main_test.py

```diff
@@ -259,33 +249,20 @@ def test_pii_files_loaded(self, mock_check_cron):
         self.assertSetEqual(set(expected_results), set(r['results']))
 
     @mock.patch('api_util.check_cron')
-    def _test_html_report_five_person(self, mock_check_cron):
+    def test_html_report_five_person(self, mock_check_cron):
         # Not sure this test is still relevant (see hpo_report module and tests)
         # TODO refactor or remove this test
         folder_prefix = '2019-01-01/'
         for cdm_file in test_util.FIVE_PERSONS_FILES:
             test_util.write_cloud_file(self.hpo_bucket,
                                        cdm_file,
                                        prefix=folder_prefix)
-        # achilles sometimes fails due to rate limits.
-        # using both success and failure cases allow it to fail gracefully until there is a fix for achilles
-        with open(test_util.FIVE_PERSON_RESULTS_FILE, 'r') as f:
-            expected_result_achilles_success = self._remove_timestamp_tags_from_results(
-                f.read())
-        with open(test_util.FIVE_PERSON_RESULTS_ACHILLES_ERROR_FILE, 'r') as f:
-            expected_result_achilles_failure = self._remove_timestamp_tags_from_results(
-                f.read())
-        expected_results = [
-            expected_result_achilles_success, expected_result_achilles_failure
-        ]
         main.app.testing = True
         with main.app.test_client() as c:
             c.get(test_util.VALIDATE_HPO_FILES_URL)
             actual_result = test_util.read_cloud_file(
                 self.hpo_bucket, folder_prefix + common.RESULTS_HTML)
-            actual_result_file = self._remove_timestamp_tags_from_results(
-                StringIO(actual_result).getvalue())
-            self.assertIn(actual_result_file, expected_results)
+        # print(actual_result)
```

Comment:
nit: better to just remove commented out code

Reply:
I was using this as a debugging spot to see the result in a browser, but I think I should add a html parser to check for the added table.